torch_xla/core/xla_model.py

```diff
@@ -852,6 +852,8 @@ def get_stablehlo(tensors=None):
   For inference graph, it is recommended to pass the model outputs to `tensors`.
   For training graph, it is not straightforward to identify the "outputs". Using empty `tensors` is recommended.
 
+  To enable source line info in StableHLO, please set env var XLA_HLO_DEBUG=1.
```

Comment:
can we add this function to https://github.com/pytorch/xla/blob/master/docs/source/index.rst#xla_model as well?

Reply:
Yes, let me add it.